File: cspro-dev/cspro/CSEntryDroid/app/src/main/java/gov/census/cspro/engine/PathException.java
package gov.census.cspro.engine;

public class PathException extends Exception {

    /**
	 * 
	 */
	private static final long serialVersionUID = 6990687125482571831L;

	public PathException(String message) {
        super(message);
    }

    public PathException(String message, Throwable throwable) {
        super(message, throwable);
    }

}

File: src/commonMain/kotlin/gov/census/cspro/exception/PathException.kt
package gov.census.cspro.exception

/**
 * Exception thrown for path-related errors
 * Ported from Android PathException.java
 */
class PathException : Exception {
    constructor() : super()
    constructor(message: String) : super(message)
    constructor(message: String, cause: Throwable) : super(message, cause)
    constructor(cause: Throwable) : super(cause)
}
